packages/edit-site/src/components/sidebar-dataviews/style.scss

```diff
@@ -6,3 +6,7 @@
 		text-transform: uppercase;
 	}
 }
+
+.edit-site-sidebar-navigation-screen-dataviews__ellipsis-button {
+	display: flex;
```

Comment:
cc @ciampo If you place a `small` button inside a `DropDownMenu` then `.components-dropdown-menu` renders at a peculiar size. Here is was 26.5px tall, in Storybook it's 27.5px. Seems like a bug to fix separately, then we can remove this style.

Reply:
Opened https://github.com/WordPress/gutenberg/pull/56005
 to fix this at the component level, you should be able to remove the custom styles once that gets merged

Reply:
New attempt in #56136